src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,231 @@
+import React, { forwardRef, useRef } from "react";
+import PropTypes from "prop-types";
+import { COLORS_BASIC, SIZES_BASIC } from "../../constants";
+import useMergeRefs from "../../hooks/useMergeRefs";
+import "./Slider.scss";
+import { bem, ensureDefaultValue } from "./SliderHelpers";
+import { SliderProvider } from "./SliderContext";
+import SliderBase from "./SliderBase/SliderBase";
+import SliderInfix from "./SliderInfix";
+
+const Slider = forwardRef(
+  (
+    {
+      ariaLabel,
+      ariaLabelledby,
+      className,
+      color,
+      "data-testid": dataTestId,
+      disabled,
+      id,
+      max,
+      min,
+      onChange,
+      ranged,
+      step,
+      showValue,
+      size,
+      value,
+      defaultValue,
+      valueFormatter,
+      valueText,
+      // ------ Additional subcomponents' props
+      indicateSelection,
+      prefix,
+      postfix
+    },
+    ref
+  ) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    const providerProps = {
+      ariaLabel,
+      ariaLabelledby,
+      color,
+      dataTestId,
+      disabled,
+      max,
+      min,
+      onChange,
+      ranged,
+      showValue,
+      size,
+      step,
+      value,
+      defaultValue: ensureDefaultValue({ defaultValue, min, max, ranged }),
+      valueFormatter,
+      valueText,
+      infixOptions: {
```

Comment:
creating a new object reference will trigger rerender of the Infix.Provider every render

Reply:
Looks like missed it after some changes/refactoring 🤦🏻‍♂️ 😆
Good catch!
Changing....

Reply:
Missed it...
Changing ...

Reply:
Done!